localization/react-intl/src/app/components/article/ExplainerForm.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "id": "explainerForm.saveSuccess",
+    "description": "Save item action success message",
+    "defaultMessage": "Article created successfully!"
```

Comment:
I'm noticing both "Explainer" and "Claim and Fact-Check" success messages are identical and currently read "Article created successfully!", if this is the case they could have the same id to avoid duplicated translatables e.g. "article.createSuccess" otherwise they could be type-specific success messages. Defer copy decisions to @brianfleming

Reply:
Yeah, I put it in as article to reduce complexity and have less changes for us when we add our next article type. In the future if we want custom messages per type we can add it later